### ISSUE TITLE
Error (org-ql-select): Not an Org buffer: *scratch*

### ISSUE THREAD

**Issue by morgandavidson** [unhammer/org-upcoming-modeline#18]
I receive the following message each time I start up Emacs: `Error (org-ql-select): Not an Org buffer: *scratch*`. `org-upcoming-modeline-mode` is fully functional though. Details on my config can be found [here](https://github.com/unhammer/org-upcoming-modeline/issues/14).

**Comment by unhammer** [comment]
Thanks for the report, and sorry it took so long (let me know if the fix didn't help, though I'm fairly sure this is because agenda-files isn't populated on first startup)

**Comment by morgandavidson** [comment]
Thanks for the fix. The issue wasn't bothering me since I asked Emacs not to show it after starting up. 

Before testing your update I tried to have the error message displayed with the previous version (20230809.957), which I still have installed, so I cleaned up `.emacs`, only leaving:

```
(require 'org)
(org-babel-load-file "PATH/emacs-init.org")
```

But I had no error message... My guess is that it is due to the Emacs update to 29.1 (when I had the issue, I used a pre-release version of 29).

I have no clue how to help you find out the cause of the issue or its resolution.
